packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -81,22 +103,12 @@ class InteractiveGraphEditor extends React.Component<Props> {
     static widgetName = "interactive-graph";
 
     static defaultProps: DefaultProps = {
-        labels: ["x", "y"],
-        range: [
-            [-10, 10],
-            [-10, 10],
-        ],
-        step: [1, 1],
+        ...InteractiveGraph.defaultProps,
         valid: true,
         backgroundImage: defaultBackgroundImage,
-        markings: "graph",
-        showProtractor: false,
-        showRuler: false,
         showTooltips: false,
-        rulerLabel: "",
-        rulerTicks: 10,
         correct: {
-            type: "linear",
+            type: InteractiveGraph.defaultProps.graph.type,
```

Comment:
Using the actual graph widget's default graph type instead of re-defining it here.